app/Http/Controllers/StatementController.php

```diff
@@ -405,7 +405,7 @@ public function storeStatement(Request $request, Validate $validate)
 
             $statement->save();
             $livecamp = Camp::getLiveCamp($filters);
-            $link = config('global.APP_URL_FRONT_END') . '/statement/history/' . $statement->topic_num . '/' . $statement->camp_num;
+            $link = '/statement/history/' . $statement->topic_num . '/' . $statement->camp_num;
```

Comment:
Are you sure it'll work?

Reply:
yes it is working and it is already used in many places